src/NearOrg/DataAvailabilityPage.jsx

```diff
@@ -230,6 +235,12 @@ const LogoLinks = styled.div`
   }
 `;
 
+const LinkLogo = styled.a`
+  img {
+    filter: ${(p) => p.$grayscale ?? "none"};
```

Comment:
_"Why do we need this filter prop here?"_

Well, the Particle Network logo itself is very colorful. This hack + change at line 327 makes it dark grey easily